app/repositories/events/EventsConfigRepository.scala

```diff
@@ -123,10 +138,20 @@ trait EventsConfigRepository {
       )
       eventItemFuture.recoverWith {
```

Comment:
Although this is always throwing an exception (if PartialFunction matches of course), I would argue that this should be ```recover``` not ```recoverWith``` - lets discuss